net/nfdump/files/nfcapd.init

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2009-2012 OpenWrt.org
+
+START=60
+USE_PROCD=1
+PROG=/usr/bin/nfcapd
+
+validate_nfcapd_section() {
+	uci_validate_section nfcapd nfcapd "${1}" \
+		'enabled:bool' \
+		'port:string' \
+		'logdir:string'
+}
+
+start_service() {
+	validate_nfcapd_section nfcapd || {
+		echo "validation failed"
+		return 1
+	}
+
+	if [ "$enabled" -gt 0 ]; then
+		procd_open_instance
+		procd_set_param stdout 1
+		procd_set_param stderr 1
+		procd_set_param command "$PROG" -S 1 -P /var/run/nfcapd.pid -p "$port" -l "$logdir"
+		procd_close_instance
```

Comment:
I'd add

procd_set_param pidfile "$pidfile"

Reply:
Actually, make sure to runtime test a change like that. I did with miniupnpd and it failed.